src/App.vue

```diff
@@ -179,15 +352,32 @@ export default {
             user: 'root',
             active: false,
             tested: false,
-        }
+        },
+        showDialogCreate: false,
+        dialogTitle: '',
+        showAddInput: '',
+        dataType: '',
+        resetSelected: 'Select',
+        index: 0,
     }),
 
     computed: {
         tablesFiltered() {
             return this.tables.filter(
-                table =>
-                    !this.tableFilter || table.indexOf(this.tableFilter) !== -1
+                table => !this.tableFilter || table.indexOf(this.tableFilter) !== -1
```

Comment:
Check if you need this `! this.tableFilter` inside of filter, because you can check before the `.filter()` and return the entire `tables`

Reply:
i would simplify this by:

```
return this.tableFilter
    ? this.tables.filter(table => table.includes(this.tableFilter))
    : this.tables;
```

Avoid indexOf, not necessary anymore... .includes or .contains does the same and already return bool